_includes/who-we-are.html

```diff
@@ -7,21 +7,7 @@
       <div class="inner">
         <h2>Yevgeniy (Jim) Brikman</h2>
         <p>
-          <a href="http://www.ybrikman.com/?ref=gruntwork-bio" target="_blank">Yevgeniy (Jim) Brikman</a> is the
-          founder of <a href="{{ site.url }}">Gruntwork</a> and
-          <a href="http://www.atomic-squirrel.net/?ref=gruntwork-bio" target="_blank">Atomic Squirrel</a>. He is the
-          author of <a href="http://www.hello-startup.net/?ref=gruntwork-bio" target="_blank">Hello, Startup</a>, a
-          comprehensive guide to building products, technologies, and teams in a startup environment. Jim has more than
-          a decade of experience building infrastructure and products at LinkedIn, TripAdvisor, Cisco Systems, and
-          Thomson Financial. He's also active in the
-          <a href="https://github.com/brikis98/" target="_blank">open source community</a>,
-          maintains a
-          <a href="http://www.ybrikman.com/writing/?ref=gruntwork-bio" target="_blank">popular blog</a>,
-          and has given
-          <a href="http://www.ybrikman.com/speaking/?ref=gruntwork-bio" target="_blank">talks</a>
-          at conferences all over the world. See
-          <a href="http://www.ybrikman.com/?ref=gruntwork-bio">ybrikman.com</a>
-          for more info.
+          Yevgeniy (Jim) Brikman is a co-founder of Gruntwork. He is the author of Hello, Startup, a 600-page book on building products, technologies, and teams rated 4.9 / 5 stars on Amazon over 20 reviews. Jim has more than a decade of experience building infrastructure and products, in some cases serving millions of users, at LinkedIn, TripAdvisor, Cisco Systems, and Thomson Financial. He's also active in the open source community, maintains a popular blog, and has given talks at conferences all over the world.
```

Comment:
s/in some cases serving millions of users/serving hundreds of millions of users/